---
title: Bloco ' Catch ' nunca atingido porque ' <name1> ' herda de ' <name2> '
ms.date: 07/20/2015
f1_keywords:
- BC42029
- vbc42029
helpviewer_keywords:
- BC42029
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
ms.openlocfilehash: 560beddd1e9051c43d610fe893a55baa2a5c559a
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 06/04/2020
ms.locfileid: "84395643"
---
# <a name="catch-block-never-reached-because-name1-inherits-from-name2"></a>Bloco ' Catch ' nunca atingido porque ' \<name1> ' herda de ' \<name2> '
Um `Catch` bloco dentro do código nunca é atingido.  
  
 Por padrão, esta mensagem é um aviso. Para obter mais informações sobre como ocultar avisos ou tratar avisos como erros, consulte [Configurando avisos no Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).  
  
 **ID do erro:** BC42029  
  
## <a name="to-correct-this-error"></a>Para corrigir este erro  
  
- Remova o bloco incorreto `Catch` .  
  
## <a name="see-also"></a>Confira também

- [Como usar o bloco try/catch para capturar exceções](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [Instrução Try...Catch...Finally](../language-reference/statements/try-catch-finally-statement.md)
